views/faq.ejs

```diff
@@ -1,42 +1,164 @@
 <div class="col-xs-10 col-xs-offset-1 col-md-8 col-md-offset-2">
   <h1>SHINE FAQ</h1>
 
-  <p>Hey, you! Thanks for coming to our FAQ page. We're here to answer any questions you have. If you don't see your answer below, email us with your <strong>full name and phone number</strong> at <a href="mailto:help@shinetext.com" target="_blank">help@shinetext.com</a> and we'll get back to you as soon as possible.</p>
+  <p>Hey, you! Thanks for coming to our FAQ page. We're here to answer any questions you have. If you don't see your answer
+    below, email us with your
+    <strong>full name and phone number</strong> at
+    <a href="mailto:help@shinetext.com" target="_blank">help@shinetext.com</a> and we'll get back to you as soon as possible.</p>
   <p>Thanks for your interest in Shine!</p>
 
   <h3>What's this Shine thing all about?</h3>
   <p>Shine makes it easier for you to take care of yourself.</p>
-  <p>Every morning, we'll send you a conversational experience around your wellbeing, with research-backed content around confidence, daily happiness, mental health and productivity.</p>
+  <p>Every morning, we'll send you a conversational experience around your wellbeing, with research-backed content around confidence,
+    daily happiness, mental health and productivity.</p>
```

Comment:
``` <p>Shine is available via SMS, Facebook Messenger, Kik, and our iOS app is available in the App Store. </p> ``` 
missing

Reply:
Ah good catch. I'll add that in before merge.